alternator/executor.hh

```diff
@@ -221,7 +222,9 @@ private:
     friend class rmw_operation;
 
     static void describe_key_schema(rjson::value& parent, const schema&, std::unordered_map<std::string,std::string> * = nullptr);
-
+    future<rjson::value> fill_table_description(schema_ptr schema, table_status tbl_status, service::client_state& client_state, tracing::trace_state_ptr trace_state, service_permit permit);
+    future<executor::request_return_type> create_table_on_shard0(service::client_state&& client_state, tracing::trace_state_ptr trace_state, rjson::value request, bool enforce_authorization);
```

Comment:
I'm always saddened to see internal implementation details that nobody wants to use from other files (it's only used inside executor.cc) reaching a header file. Which is why I am asking for an explanation for this change.